edb/api/errors.txt

```diff
@@ -112,6 +112,8 @@
 0x_05_01_00_00   InvalidValueError
 0x_05_01_00_01   DivisionByZeroError
 0x_05_01_00_02   NumericOutOfRangeError
+0x_05_01_00_03   InvalidLimitError
```

Comment:
Is it worth adding new error types for this? We could just use InvalidValueError directly? (Does adding error types require updating the bindings?)

Reply:
Probably not, because we are mulling possibly adding support for negative `LIMIT`/`OFFSET` (see #211).

Reply:
Alright, so just make it an `InvalidValueError`?

Reply:
Yeah, that seems reasonable.